### PR TITLE
Update ES Forward Documentation

### PR DESCRIPTION
**Pull request by Clee2691**
### Description
This PR fixes the documentation for forwarding to Elasticsearch.

/cc @cahartma
/assign @jcantrill 

/cherry-pick master

### CODE REVIEW

**Comment by jcantrill** [comment]
cc @Seth-Karlo

File: docs/features/logforwarding/outputs/elasticsearch-forwarding.adoc [hunk 1]
```diff
@@ -51,5 +52,5 @@ spec:
...
 ----
 +
-<1> For an `https` prefix, specify the name of the secret required by the endpoint for TLS communication. The secret must exist in the `openshift-logging` project, and must have keys of: `tls.crt`, `tls.key`, and `ca-bundle.crt` that point to the respective certificates that they represent. Otherwise, for `http` and `https` prefixes, you can specify a secret that contains a username and password. 
-<2> Forwarding to an external Elasticsearch of version 8.x or greater requires the `Version` field to be specified otherwise this can be omitted.
\ No newline at end of file
+<1> Forwarding to an external Elasticsearch of version 8.x or greater requires the `version` field to be specified otherwise this can be omitted. The `version` field is nested under `elasticsearch`.
+<2> For an `https` prefix, specify the name of the secret required by the endpoint for TLS communication. The secret must exist in the `openshift-logging` project, and must have keys of: `tls.crt`, `tls.key`, and `ca-bundle.crt` that point to the respective certificates that they represent. Otherwise, for `http` and `https` prefixes, you can specify a secret that contains a username and password.
```
[the remark this answers is not in the document]
Comment (Clee2691) [reply docs/features/logforwarding/outputs/elasticsearch-forwarding.adoc:56]: I took this line from the official logging docs linked here: 

[Forward ES](https://docs.openshift.com/container-platform/4.12/logging/cluster-logging-external.html#cluster-logging-collector-log-forward-es_cluster-logging-external), see bullet point #7

If this isn't right, what should it be?

[the remark this answers is not in the document]
Comment (anpingli) [reply docs/features/logforwarding/outputs/elasticsearch-forwarding.adoc]: That is basic configuration for all supported output plugin, no matter which version it is.

[the remark this answers is not in the document]
Comment (Clee2691) [reply docs/features/logforwarding/outputs/elasticsearch-forwarding.adoc]: Ok. So it looks like the official docs with also need to be updated with this description.

**Comment by Clee2691** [comment]
Closing this to base this change on master and cherry pick it back into release-5.7

**Pull request closed without merging**